tur-multilib/winetricks/build.sh

```diff
@@ -0,0 +1,74 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Winetricks/winetricks
+TERMUX_PKG_DESCRIPTION="Work around problems and install applications under Wine"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+_COMMIT=a9a556719e4fd28fc5984a963e2d666ec809f554
+_COMMIT_DATE=20240216
+_COMMIT_TIME=071709
+TERMUX_PKG_VERSION="0.0.20240216.071709"
```

Comment:
Is there any reason that winetricks should use the latest git version instead of the Release version?

Reply:
Your questioning is right. Originally I found this official guide (https://wiki.winehq.org/Winetricks) to install the latest git version. But it seems that most distro distributes the stable release version. Let me commit a change to the stable release.